src/styles/navbar.css

```diff
@@ -69,28 +93,124 @@ header {
       display: block;
       font-size: 1.5rem;
       cursor: pointer;
+      &:hover {
+        color: var(--secondary-color);
+      }
+      &:active {
+        color: var(--secondary-color-dark);
+        scale: 0.9;
+      }
     }
     .nav-links {
-      height: 0;
-      overflow: hidden;
-      position: absolute;
-      top: 4rem;
-      left: 0;
-      width: 100%;
-      background-color: #333;
-      transition: all 0.3s ease-in-out;
-      ul {
-        flex-direction: column;
-        .nav-link {
-          margin: 0;
-          padding: 1rem;
-          width: 100%;
-          text-align: center;
-        }
+      display: none;
+    }
+  }
+}
+.hamburger-wrapper {
+  visibility: hidden;
```

Comment:
This property is probably not able to be animated, which means it will be visible or not you can not fade it. Not sure if you want to fade this in and out, if you would want to do this you could use `opacity`.

Reply:
I am using a combination of opacity and visibility for this and this is the only way I made it work. Maybe we could have a quick look later.